source/Playnite.DesktopApp/Controls/SettingsSections/AppearanceAdvanced.xaml

```diff
@@ -28,155 +31,270 @@
+                        </DataTemplate>
+                    </ComboBox.ItemTemplate>
+                </ComboBox>
+            </GridEx>
 
-        <GridEx ColumnCount="2" RowCount="2" Margin="0,20,0,0">
-            <ContentControl Content="{DynamicResource LOCSettingsTextRenderingMode}" Grid.Column="0" Grid.Row="0"
-                            Margin="0,10,10,10" VerticalAlignment="Center"
-                            ContentStringFormat="{}{0} *"/>
-            <ComboBox Grid.Column="1" Grid.Row="0" VerticalAlignment="Center" MinWidth="100"
-                      SelectedValue="{Binding Settings.TextRenderingMode}">
-                <ComboBox.ItemsSource>
-                    <x:Array Type="{x:Type sys:Enum}">
-                        <p:TextRenderingModeOptions>Auto</p:TextRenderingModeOptions>
-                        <p:TextRenderingModeOptions>Aliased</p:TextRenderingModeOptions>
-                        <p:TextRenderingModeOptions>Grayscale</p:TextRenderingModeOptions>
-                        <p:TextRenderingModeOptions>ClearType</p:TextRenderingModeOptions>
-                    </x:Array>
-                </ComboBox.ItemsSource>
-                <ComboBox.ItemTemplate>
-                    <DataTemplate>
-                        <TextBlock Text="{Binding Converter={pcon:ObjectToStringConverter}}"/>
-                    </DataTemplate>
-                </ComboBox.ItemTemplate>
-            </ComboBox>
-
-            <ContentControl Content="{DynamicResource LOCSettingsTextFormattingMode}" Grid.Column="0" Grid.Row="1"
-                            Margin="0,10,10,10" VerticalAlignment="Center"
-                            ContentStringFormat="{}{0} *"/>
-            <ComboBox Grid.Column="1" Grid.Row="1" VerticalAlignment="Center" MinWidth="100"
-                      SelectedValue="{Binding Settings.TextFormattingMode}">
-                <ComboBox.ItemsSource>
-                    <x:Array Type="{x:Type sys:Enum}">
-                        <p:TextFormattingModeOptions>Ideal</p:TextFormattingModeOptions>
-                        <p:TextFormattingModeOptions>Display</p:TextFormattingModeOptions>
-                    </x:Array>
-                </ComboBox.ItemsSource>
-                <ComboBox.ItemTemplate>
-                    <DataTemplate>
-                        <TextBlock Text="{Binding Converter={pcon:ObjectToStringConverter}}"/>
-                    </DataTemplate>
-                </ComboBox.ItemTemplate>
-            </ComboBox>
-        </GridEx>
-
-        <TextBlock Text="{DynamicResource LOCSettingsTextRenderingNotice}"
-                   TextWrapping="Wrap" Margin="0,10,0,0"/>
-    </StackPanel>
+            <TextBlock Text="{DynamicResource LOCSettingsTextRenderingNotice}"
+                       TextWrapping="Wrap" Margin="0,10,0,0"/>
+
+            <TextBlock Text="{DynamicResource LOCSettingsDatesFormatsLabel}" Margin="0,15,0,0" />

```

Comment:
Add `*` here so it's clear that these changes require restart.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3024/commits/964f0a5b948da268ddebab22fe5dbd1adfd3ee6d